stories/greyVest/decorator.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
-import greyVest from '../../src/themes/greyVest'
-import { ThemeProvider } from '../../src/utils/theme'
+import { Fonts, Style } from '../../src/greyVest'
```

Comment:
it might be cleaner to just import `Root` and do

```js
export default Story => <Root><Story /></Root>
```

in case we add other things besides `Fonts` and `Style`

Reply:
this would be much cleaner - the only thing is that Root has the GreyVest theme styles in it as well as the component styles. I figured we specifically _don't_ want those, to make sure the lib looks okay on its own